buildpipeline/pipeline.json

```diff
@@ -49,6 +49,23 @@
           }
         },
         {
+          "Name": "DotNet-CoreFx-Trusted-Linux",
+          "Parameters": {
+            "PB_DockerTag": "alpine-3.6-3148f11-20171119021156",
+            "PB_BuildArguments": "-buildArch=x64 -Release -stripSymbols -RuntimeOS=alpine.3.6 -- /p:PortableBuild=false",
+            "PB_BuildTestsArguments": "-buildArch=x64 -Release -SkipTests -Outerloop -RuntimeOS=alpine.3.6 -- /p:ArchiveTests=false /p:EnableDumpling=true /p:PortableBuild=false",
+            "PB_SyncArguments": "-p -RuntimeOS=alpine.3.6 -- /p:ArchGroup=x64 /p:PortableBuild=false",
+            "PB_TargetQueue": "Alpine.3.6.Amd64",
+            "PB_CreateHelixArguments": "/p:EnableCloudTest=false /p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:TestProduct=corefx /p:TimeoutInSeconds=1200 /p:TargetOS=Linux"
```

Comment:
This will work, but for the condition added to the build step to work, this needs to be in its own property, for example:

```
"EnableCloudTest" : "false",
"PB_CreateHelixArguments": "/p:EnableCloudTest=$(EnableCloudTest) /p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:TestProduct=corefx /p:TimeoutInSeconds=1200 /p:TargetOS=Linux"
```

Reply:
Ok, so I assume that the `"condition": "ne(variables['PB_EnableCloudTest'], 'false')", ` would then be `"condition": "ne(variables['EnableCloudTest'], 'false')", ` instead, right?

Reply:
Or should I rather use the `PB_` variant?

Reply:
I'd defer to @chcosta, it doesn't super matter but I think it's unnecessary here.   In some cases, this matters specifically when the property is provided at the pipebuild.exe level (hence "pb_") because those are global properties that can be accidentally sucked up into builds they weren't meant to.  In the case of EnableCloudTest it's never going to be set globally (or all text execution gets disabled, maybe we'd want to do that... doesn't change my answer) so it doesn't technically need PB_, but I don't care if you use it.